src/ledger/secp256k1.ts

```diff
@@ -83,8 +83,8 @@ export class Secp256k1PublicKey implements PublicKey {
     return rawKey;
   }
 
-  private readonly rawKey: ArrayBuffer;
-  private readonly derKey: DerEncodedPublicKey;
+  readonly rawKey: ArrayBuffer;
```

Comment:
Is it safe to expose those publically?

Reply:
This implements `publicKey` and it's public there...

Reply:
I see. I'm still a bit confused about why making those public given that functions `toDer()` and `toRaw()` are already exposing those values. So it feels like redundant but, just saying.